### ISSUE TITLE
Feedback requested - Should we move the "Discussion Forum" away from Google Groups 

### ISSUE THREAD

**Issue by rbygrave** [ebean-orm/ebean#1252]
## As I understand it:
- Some people are **blocked from using Google services** (i.e. Developers in China) and specifically are blocked from using Google Groups.
- Some people would prefer not to use Google services for personal reasons

I am requesting feedback on this question, especially from people who are blocked from using Google Groups.  If that is you and you would like "discussion forums" moved please provide feedback on this ticket.

## Project policy 
This is the policy for this project as per: http://ebean-orm.github.io/support

In short it is good to **keep questions and discussions OUT of github issues**.  That we should leave github issues for tickets that result in code changes (Bugs, Enhancements, anything that results in a git commit basically).

I am breaking the policy raising this 'discussion ticket'. That is because I believe some people can't and won't partake in this discussion in our Google group forum. So we are making an exception here by logging this as a github issue for discussion.

Thanks, Rob.

**Comment by corrspt** [comment]
Hi Rob,

First of all, I really appreciate your work on Ebean, I really like using it.

Regarding the forums, one of the things I find is that searching on google for "ebean related stuff" very rarely leads me to the Google Group (I just tried with the exact topic name and it didn't show up). Which I think is very bad. Not sure which platform would be the most valuable though.

I might just be my ignorance of how google groups work, by I would like to have some "following/subscribe" (just like I do here on github issues) to keep track of the questions/issues the community has.

I understand that github issues are a bad place for the discussions, but I've been saved so many times from issues with other libraries because google does a great job at indexing github issue (not suggesting to use them for discussion, just pointing out the effect).

**Comment by rbygrave** [comment]
> google groups work, by I would like to have some "following/subscribe" 

We should be able to choose from a few options there - a digest email once a day vs every post etc, so have a look for those options.

> searching on google for "ebean related stuff" very rarely leads me to the Google Group 

Ah interesting yes.

Well, this post didn't draw out much comment from people who I thought were blocked (aka developers in China) so I guess they can't be too fussed.

In general I think I'll close this issue for now and we can ponder again. Long term I think I'm keen on finding another option but short and medium term there are other things to do and it seems there is no pressing need.

Closing for now.
